### PR TITLE
New version: Schrodinger.PyMOL version 3.1.1

### PR DESCRIPTION
**Pull request by spectopo**
Created by [🥟 Dumplings](https://github.com/SpecterShell/Dumplings) in workflow run [#9529](https://github.com/SpecterShell/Dumplings/actions/runs/11827298579).

<details>

<summary>Log</summary>

````
Updated: 3.1.0 → 3.1.1
Submitting WinGet manifests
Creating manifests
Uploading manifests and making commits
Creating a pull request
````

</details>

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/191294)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
Manual Validation is underway. This occurred:
image

**Review by ItzLevvie — APPROVED** [review]

File: manifests/s/Schrodinger/PyMOL/3.1.1/Schrodinger.PyMOL.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- mol2
+- pdb
+- pqr
+- pse
+- pse.gz
+- pze
+- sdf
+- xyz
+ReleaseDate: 2024-11-13
+Installers:
```
Comment (stephengillie) [inline manifests/s/Schrodinger/PyMOL/3.1.1/Schrodinger.PyMOL.installer.yaml:21]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
image

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by stephengillie** [comment]
This package installs and launches normally in a Windows 10 VM.

**Comment by stephengillie** [comment]
This package installs and launches normally in a Windows 10 VM.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**